RS3NXTRefactorer.java

```diff
@@ -1393,4 +1396,46 @@ public String toString() {
             return "ServerProt[opcode="+opcode+", size="+size+", name="+name+", addr= "+addr+" ]";
         }
     }
+    
+    /**
+     * Used to print out and int array of packet sizes and a map of opcode names.
+     * This is to make updating new revision opcodes faster.
+     *
+     * TODO: When the core packet class is finished, this will print out the whole
+     *  Kotlin file to make moving to different revisions as painless as possible
+     */
+    private void printOpenRS3PacketFormat() {
```

Comment:
I actually wrote something that does this earlier, it maps the `ServerProt` names and sizes, since `Incoming` / `Outgoing` can be quite ambiguous (incoming on the client is serverprot, incoming on the server is clientprot).

I've attached the code I used below, perhaps you could replace the code with that?
```java
        printf("%n// Packet names for ServerProt (server to client), mapped opcode->name%n");
        printf("val SERVER_PROT_NAMES = mapOf(%n");
        for(int i = 0; i < packets.length; i++) {
            ServerProtInfo info = packets[i];
            if (info.name == null) continue;

            printf("    %d to \"%s\",%n", info.opcode, info.name);
        }
        printf(")%n");

        printf("%n// Packet sizes for ServerProt (server to client), mapped opcode->size%n");
        printf("val SERVER_PROT_SIZES = mapOf(%n");
        for(int i = 0; i < packets.length; i++) {
            ServerProtInfo info = packets[i];
            if (info.name == null) continue;

            printf("    %d to %d,%n", info.opcode, info.size);
        }
        printf(")%n");```